server/service/src/sync/test/test_data/invoice_line.rs

```diff
@@ -99,24 +99,26 @@ fn trans_line_1_pull_record() -> TestSyncPullRecord {
 fn trans_line_1_push_record() -> TestSyncPushRecord {
     TestSyncPushRecord {
         change_log: ChangelogRow {
-            id: 2,
+            cursor: 2,
```

Comment:
I had to touch invoice_line translator in downstream PR, and though to outsource renaming to serde